src/constants/appRpcProvider.ts

```diff
@@ -0,0 +1,156 @@
+import { Network } from '@ethersproject/networks'
+import { JsonRpcProvider, Provider } from '@ethersproject/providers'
+
+function now() {
+  return new Date().getTime()
+}
+
+function checkNetworks(networks: Array<Network>): Network | null {
+  let result: Network | undefined = undefined
```

Comment:
nit: This would let you remove the `?? null` in the return at the bottom
```suggestion
  let result: Network | null = null
```